### PR TITLE
Prepare gottcha2 for bioconda release

### PR DESCRIPTION
**Pull request by chernikovanna**
- add database pull script
- package with setup.py

### CODE REVIEW

**Comment by poeli-lanl** [comment]
Consider including `install_requires` in the setup.

Also, I suggest we should specify a package and an entry point for profiling and pulling databases, like [program] [command] [arguments]. For example:
>gottcha2 profile --help
>gottcha2 pull --help

How do you think?

**Review by poeli-lanl — CHANGES_REQUESTED** [review]
Suggest adding `install_requires` and provide commands instead of running separate scripts.

**Comment by chernikovanna** [comment]
> Consider including `install_requires` in the setup.
> 
> Also, I suggest we should specify a package and an entry point for profiling and pulling databases, like [program] [command] [arguments]. For example:
> 
> > gottcha2 profile --help
> > gottcha2 pull --help
> 
> How do you think?

I think that's doable! I started building the entry point and I updated our CI but I have a bit more debugging to do.

File: .github/workflows/gottcha.yml [hunk 1]
```diff
@@ -25,12 +25,15 @@ jobs:
...
 
-      - name: Check version
+      - name: Check Install
         run: |
           conda list
-          GOTTCHA2/gottcha2.py --version
+          cd GOTTCHA2
+          python setup.py install
+          gottcha2 profile --help
+          gottcha2 pull --help
```
Comment (poeli-lanl) [inline .github/workflows/gottcha.yml:34]: test pulling a small gottcha_db database

**Pull request merged**